sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -430,6 +435,28 @@ public void removeContexts(final @NotNull String key) {
     contexts.remove(key);
   }
 
+  /**
+   * Returns the Scopes's attachments
+   *
+   * @return the attachments
+   */
+  @ApiStatus.Experimental
```

Comment:
L: fine keeping it, but it's a package-private method, so it's not supposed to be used anyway.

Reply:
I'm going to remove it.